### DISCUSSION TITLE
Qualyfing Lap compare over two Year

### DISCUSSION THREAD

**Discussion by cecereroberto** [theOehrly/Fast-F1#352] [Q&A]
Hi,
Can anyone tell me if with. Fast-F1 it is possible to graphically compare a qualifying lap on the same track in two different years. For example, the qualification of Leclerc in bahrain 2022 with that of 2023? thank you in advance for the support

**Comment by theOehrly** [comment]
Yes, it is possible. I recently saw a tweet where someone did just that.

**Comment by cecereroberto** [comment]
I’ve tried to find an example in Twitter without success :-(Inviato da iPhoneIl giorno 14 apr 2023, alle ore 19:25, Philipp Schaefer ***@***.***> ha scritto:﻿
Yes, it is possible. I just saw a tweet where someone did just that.

**Reply by theOehrly** [reply]
https://twitter.com/NaturalParadigm/status/1631851405777485825

Done through F1Tempo, but there's nothing stopping you from doing it with FastF1 directly.
